modules/configuration/php/subproject.class.inc

```diff
@@ -51,9 +51,9 @@ class Subproject extends \NDB_Form
 
         $subprojectList = \Utility::getSubprojectList();
         $subprojects    = array();
-        foreach ($subprojectList as $subprojectID => $title) {
+        foreach ($subprojectList as $subprojectID) {
```

Comment:
same comment as above, this change switches from key assignment to value assignment to the same variable